core/modules/admin_bar/admin_bar.theme.inc

```diff
@@ -5,6 +5,15 @@
  */
 
 /**
+ * Renders an alert icon to display in the administration bar.
+ *
+ * @ingroup themeable
+ */
+function theme_admin_bar_alert_icon($variables) {
+  return '<img class="admin-bar-alert-icon" src="' . $variables['src'] . '" width="14" height="12" alt="' . $variables['alt'] . '" />';
```

Comment:
does this have to be an IMG tag?  We can't use a span with CSS? I should go read the issue...